src/SprykerEco/Zed/Unzer/Business/Oms/Command/ChargeOmsCommand.php

```diff
@@ -0,0 +1,45 @@
+<?php
+
+/**
+ * MIT License
+ * For full license information, please view the LICENSE file that was distributed with this source code.
+ */
+
+namespace SprykerEco\Zed\Unzer\Business\Oms\Command;
+
+use Generated\Shared\Transfer\OrderTransfer;
+use SprykerEco\Zed\Unzer\Business\Payment\Processor\UnzerChargeablePaymentProcessorInterface;
+use SprykerEco\Zed\Unzer\Business\Payment\ProcessorResolver\UnzerPaymentProcessorResolverInterface;
+
+class ChargeOmsCommand extends AbstractOmsCommand implements UnzerOmsCommandInterface
```

Comment:
`ChargeOmsCommand` -> `ChargeUnzerOmsCommand`

Reply:
This interface `UnzerOmsCommandInterface` will be used not just for Charge command?
If not lets rename it to `ChargeUnzerOmsCommandInterface`

Reply:
It will be used not just for charge